scripts/validate_v3_contracts.py

```
@@ -0,0 +1,173 @@
+ROOT = Path(__file__).resolve().parents[1]
+FLOW_IDS = ["signal", "plan", "build", "review", "gate", "deploy", "wisdom"]
+MATURITY = {"native", "preview", "experimental", "unsupported", "unknown"}
+INSTALL_MODES = {"native", "managed-files", "local", "bundle"}
+
+
+def fail(message: str) -> None:
+    print(f"ERROR: {message}", file=sys.stderr)
+    raise SystemExit(1)
+
+
+def load_json(path: Path) -> Any:
```

Ruff (0.16.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `load_json`

(ANN401)